### PR TITLE
[jk] Update Triggers list page with filters and clean up subheader

### PR DESCRIPTION
**Pull request by johnson-mage**
# Description
- Add toolbar to Triggers list page for a pipeline with filters, and consolidate "Run pipeline now"
- Add `description`  attribute to pipeline schedule model, and display `description` in Triggers table. 

# How Has This Been Tested?
Setting filters:
triggers page update

Updated Triggers table:
image

Editing description
editing description

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [x] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: mage_ai/orchestration/db/migrations/versions/117f8c3b9c42_migration_message.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('pipeline_schedule', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('description', sa.Text(), nullable=True))
+
+    with op.batch_alter_table('secret', schema=None) as batch_op:
```
Comment (wangxiaoyou1993) [inline mage_ai/orchestration/db/migrations/versions/117f8c3b9c42_migration_message.py:24]: You don't need to alter secret table.

**Comment by wangxiaoyou1993** [comment]
did you test the db migration in both sqlite and postgres?

**Comment by johnson-mage** [comment]
> did you test the db migration in both sqlite and postgres?

@wangxiaoyou1993 Yes, I did test with both. I ran the app using both the `docker-compose.yml` and `pg-docker-compose.yml` files and checked to make sure the description columns were added in both databases. I didn't change the `sqlalchemy.url` for postgres in the `alembic.ini` file to re-run any migrations, tho. Not sure if that was necessary for testing.

**Review by wangxiaoyou1993 — APPROVED** [review]

**Comment by wangxiaoyou1993** [comment]
Could you update button text in this doc? https://docs.mage.ai/guides/blocks/dynamic-blocks#run-pipeline

**Pull request merged**